packages/@ionic/cli/src/commands/deploy/core.ts

```diff
@@ -132,7 +151,7 @@ export abstract class DeployConfCommand extends DeployCoreCommand {
       return false;
     }
     if (!plistPath) {
-      this.env.log.info(`No Capacitor iOS project found.`);
+      this.env.log.warn(`No ${chalk.bold('Capacitor iOS')} project found you will need to rerun ${chalk.yellow('ionic deploy configure')} if you add it later.`);
```

Comment:
Try not to use chalk directly--it can lead to people using different colors and be a nonsensical rainbow for users.

Instead, use the semantically named colors from `color.ts`: https://github.com/ionic-team/ionic-cli/blob/develop/packages/@ionic/cli/src/lib/color.ts

Commands are supposed to use the "input" color, which ends up being cyan, not yellow. Use "strong" for bold.

Reply:
@dwieeb updated